src/java/com/vaklinov/zcashui/ZCashClientCaller.java

```diff
@@ -1331,4 +1324,46 @@ private void decomposeJSONValue(String name, JsonValue val, Map<String, String>
 			map.put(name, val.toString());
 		}
 	}
+
+	public boolean applyZendCommands() {
+		Properties config_file = new Properties();
+
+		boolean run_option = false;
+		try {
+			File commandsFile = new File(OSUtil.getSettingsDirectory() + File.separator + "commands.conf");
+
+			if(commandsFile.exists()) {
+				config_file.load(new FileInputStream(commandsFile));
+				commands = config_file.getProperty("ZendCommands");
+				run_option = config_file.getProperty("RunOnce").trim().equals("1") ? true : false;
+
+				if (run_option) {
+					Log.info("RunOnce config file: " + run_option);
```

Comment:
In case of run once I would suggest to delete the file instead of rewriting it with empty commands. This will prevent to rewrite it every time the wallet is re-started.